### ISSUE TITLE
Fix sprintf warnings in R-devel (CRAN check)

### ISSUE THREAD

**Issue by aryoda** [aryoda/tryCatchLog#60]
See email from Prof. Brian Ripley (Oct 23, 2020):

> CRAN packages with sprintf warnings in R-devel:
>
> That is
> 
> CodeDepends DescTools FluMoDL GmAMisc LOGICOIL ODB Publish
> R.utils anytime aos aroma.core assertive.datetimes colt dbmss dbstats
> doFuture easyDes ecespa ems fastJT fun future future.BatchJobs
> future.apply future.batchtools future.callr geojsonio ggedit globals
> highlight huxtable jtGWAS jwutil knitLatex ldaPrototype lfe listenv
> logmult matlib meteR mlr mlrMBO move munsellinterpol nets
> nvmix odin openxlsx parallelly pkgmaker plfMA pomp progressr rdlocrand
> rioja salso scan sjstats smacpod spatialEco spatialsegregation
> spatstat stablelearner sybil tcpl tinyProject tinytest **tryCatchLog**
> tsdecomp vetr xgboost
> 
> R-devel has a new runtime check in sprintf() that all arguments are used 
> by the format and this is showing one or more warnings in the checks of 
> these packages, usually in the examples.  E.g.
> 
> Warning in sprintf("%s$", extension, ignore.case = TRUE) :
>    argument not used by format
> Warning in sprintf("<a href='%s'>%s</a>", getFunctionLinks(tokens[w]), 
> tokens[w],  :
>    argument not used by format
> 
> Please investigate (by running  R CMD check with the latest R-devel and 
> grepping the .Rcheck directory).  In some cases the warning seems to be 
> from another package, usually one with the same maintainer.  If from one 
> not on this list, please let CRAN and its maintainer know.  If you think 
> this is a false positive, please contact Tomas Kalibera (in Cc:).
>

**Comment by aryoda** [comment]
First analysis: The offending line may be:

https://github.com/aryoda/tryCatchLog/blob/538f3ea82fc8d0d2814699664a348da95c46b41e/R/log2console.R#L32

The trailing new line is a stupid mistake ;-)

**Comment by aryoda** [comment]
Update: CI/CD for R-devel indeed shows

```
> tryCatchLog:::log2console("WARN", "this is my last warning")
Warning in sprintf("%s [%s] %s\n", severity.level, log.time, msg, "\n") :
  one argument not used by format '%s [%s] %s
'
WARN [2020-10-27 08:44:47] this is my last warning
```

in the `tryCatchLog-Ex.Rout` file

**Comment by aryoda** [comment]
Fixed in version 1.2.0
